src/web/ScrollControls.tsx

```diff
@@ -210,6 +210,7 @@ export function ScrollControls({
   return <context.Provider value={state}>{children}</context.Provider>
 }
 
+// @ts-ignore
 const ScrollCanvas: ForwardRefComponent<{}, THREE.Group> = /* @__PURE__ */ React.forwardRef(({ children }, ref) => {
```

Comment:
Why are these here and in `Instances`?

Reply:
i updated react types, they were still react 17, and suddenly it doesn't know "children" any longer.